entrypoint.sh

```diff
@@ -12,10 +12,18 @@ if echo ${@} | grep -q "cloud-provider=azure"; then
   fi
 fi
 
+# find docker root
+if [ -f "/etc/docker/daemon.json" ]; then
```

Comment:
if this is not a file it should probably still default to /var/lib/docker?

Reply:
good catch